CHANGELOG.mdx

```diff
@@ -16,6 +16,20 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [3.6.15] - 2023-12-01
```

Comment:
the date doesnot seem to match with 3.6.x https://github.com/RasaHQ/rasa/blob/3.6.x/CHANGELOG.mdx#L19